pkgs/applications/audio/soundux/default.nix

```
@@ -0,0 +1,69 @@
+{
+  lib, stdenv, fetchFromGitHub, cmake, ninja, pkg-config, makeWrapper, wrapGAppsHook,
+  libappindicator-gtk3, openssl, pipewire, pulseaudio, webkitgtk, xorg,
+  libpulseaudio, libwnck3, makeDesktopItem,
+  downloaderSupport ? true, ffmpeg, youtube-dl
+}:
+
+let
+  downloaderPath = lib.makeBinPath [ffmpeg youtube-dl ];
+  # Soundux loads pipewire, pulse and libwnck optionally during runtime
+  dynamicLibraries = lib.makeLibraryPath [libpulseaudio pipewire libwnck3 ];
+in
+stdenv.mkDerivation rec {
+  pname = "soundux";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "Soundux";
+    repo = "Soundux";
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "15kd9vl7inn8zm5cqzjkb6zb9xk2xxwpkm7fx1za3dy9m61sq839";
+  };
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
+  dontWrapGApps = true;
+
+  installPhase = ''
+    mkdir -p $out/opt $out/bin
+    cp -r dist soundux-${version} $out/opt
+    mkdir -p $out/share/
+    ln -s "${desktopItem}/share/applications" $out/share/
+  '';
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = binaryName;
+    icon = pname;
+    desktopName = "Soundux";
+    genericName = meta.description;
+    categories = [ "Multimedia" ];
```

```suggestion
    categories = [ "Audio" ];
```
There is no such category (check [registered categories](https://specifications.freedesktop.org/menu-spec/latest/apa.html)). We can just use `Audio` here